src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Controller/ConfigurationControllerTest.php

```diff
@@ -38,7 +38,7 @@ public function setUp()
         $this->loadFixtures([]);
 
         $this->client = static::createClient();
-        $this->password = $this->client->getKernel()->getContainer()->getParameter('management_password');
+        $this->password = 'management_password_test';
```

Comment:
The ConfigurationController currently does not test whether a 400 is returned when a command is invalid.

In fact, a 500 is thrown. The `InvalidCommandExceptionListener` is disregarded because the generic  `ExceptionListener` seems to have precedence over it. 

We might want to consider converting `InvalidCommandExceptions` to `BadCommandRequestExceptions` by hand in relevant controllers. This is out of scope for this PR.